lib/reporting/identity_verification_report.rb

```diff
@@ -257,9 +283,13 @@ def data
           case event
           when Events::IDV_FINAL_RESOLUTION
             event_users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << user_id if row['identity_verified'] == '1'
-            event_users[Results::IDV_FINAL_RESOLUTION_GPO] << user_id if row['gpo_verification_pending'] == '1'
-            event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON] << user_id if row['in_person_verification_pending'] == '1'
-            event_users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << user_id if row['fraud_review_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << user_id if row['gpo_verification_pending'] != '1' && row['in_person_verification_pending'] != '1' && row['fraud_review_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_GPO] << user_id if row['gpo_verification_pending'] == '1' && row['in_person_verification_pending'] != '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_GPO_FRAUD_REVIEW] << user_id if row['gpo_verification_pending'] == '1' && row['in_person_verification_pending'] != '1' && row['fraud_review_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON] << user_id if row['gpo_verification_pending'] != '1' && row['in_person_verification_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON_FRAUD_REVIEW] << user_id if row['gpo_verification_pending'] != '1' && row['in_person_verification_pending'] == '1' && row['fraud_review_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_GPO_IN_PERSON] << user_id if row['gpo_verification_pending'] == '1' && row['in_person_verification_pending'] == '1'
+            event_users[Results::IDV_FINAL_RESOLUTION_GPO_IN_PERSON_FRAUD_REVIEW] << user_id if row['gpo_verification_pending'] == '1' && row['in_person_verification_pending'] == '1' && row['fraud_review_pending'] == '1'
```

Comment:
my eyes area glazing over reading this, are there any ways we can groups ome of the repeated stuff?
```suggestion
            fraud_review_pending = row['in_person_verification_pending'] == '1'
            gpo_verification_pending = row['gpo_verification_pending'] == '1'
            in_person_verification_pending = row['in_person_verification_pending'] == '1'
            if fraud_review_pending
              event_users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << user_id if !gpo_verification_pending && !in_person_verification_pending
              event_users[Results::IDV_FINAL_RESOLUTION_GPO_FRAUD_REVIEW] << user_id if gpo_verification_pending && !in_person_verification_pending
              event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON_FRAUD_REVIEW] << user_id if !gpo_verification_pending && in_person_verification_pending
              event_users[Results::IDV_FINAL_RESOLUTION_GPO_IN_PERSON_FRAUD_REVIEW] << user_id if gpo_verification_pending && in_person_verification_pending
            end
            event_users[Results::IDV_FINAL_RESOLUTION_GPO] << user_id if gpo_verification_pending && !in_person_verification_pending
            event_users[Results::IDV_FINAL_RESOLUTION_IN_PERSON] << user_id if !gpo_verification_pending && in_person_verification_pending
            event_users[Results::IDV_FINAL_RESOLUTION_GPO_IN_PERSON] << user_id if gpo_verification_pending && in_person_verification_pending

```

Reply:
I just pushed an update that re-arranges some of the categories (after talking it over with @artfulaction) and took a swing at grouping them some. PTAL

Reply:
love those! thank you so much